src/mount/extensions/explicit_deps.clj

```diff
@@ -36,7 +36,7 @@
   "Just like the core `start`, except with an `up-to-var`it only
   starts the explicit transitive dependencies of that state."
   ([]
-   (mount/start))
+   (start nil))
```

Comment:
The start and stop functions have been modified to rebind the states in dependency order before starting and stopping.